src/SymptomHistory/index.tsx

```diff
@@ -10,14 +17,30 @@ import { SymptomEntry } from "./symptomHistory"
 import { Text, StatusBar } from "../components"
 import { useStatusBarEffect } from "../navigation"
 import SymptomEntryListItem from "./SymptomEntryListItem"
+import Formatter from "./Formatter"
```

Comment:
Is there a better name for this? Maybe something like `FormatSymptomHistory` to be more specific?